docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,88 @@
+"""
+Coriolis parameters across the world
+====================================
+
+This code computes the Coriolis parameter for an area and stores it in a cube.
+
+It creates and plots a map showing the Coriolis parameters, overlaid on top of
+an Orthographic projection of the globe.
+
+"""
+
+import cartopy.crs as ccrs
+import iris
+from iris.coord_systems import GeogCS
+import iris.plot as iplt
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 200
+    longitudes = np.linspace(-180.0, 180.0, resolution)
+    latitudes = np.linspace(-90.0, 90.0, resolution)
+    lon2d, lat2d = np.meshgrid(longitudes, latitudes)
+
+    # Omega is the Earth's rotation rate, expressed in radians per second
+    omega = 7.29e-5
+
+    # Now to create our data
+    data = 2. * omega * np.sin(np.deg2rad(lat2d))
+
+    # We now need to define a coordinate system for the plot.
+    # Here we'll use GeogCS; 6371229 is the radius of the Earth in metres.
+    cs = GeogCS(6371229)
+
+    # The Iris coords module turns the latitude list into a coordinate array.
+    # Coords then applies an appropriate standard name and unit to it.
+    lat_coord = iris.coords.DimCoord(latitudes,
+                                     standard_name='latitude',
+                                     units='degrees',
+                                     coord_system=cs)
+
+    # The above process is repeated for the longitude coordinates.
+    long_coord = iris.coords.DimCoord(longitudes,
```

Comment:
I have a (perhaps unreasonably) strong personal preference for `lon_coord` over `long_coord`!

The following is a hand-wavy explanation for my feeling on this. I think this is for a few reasons, firstly it makes the variable name the same length as the corresponding variable for latitude, perhaps making it easier to see these two are of a kind... Secondly, `long_coord` makes me think about the length of the coordinate, not about it containing longitudes! These may be the ramblings of a mad man, but I think I genuinely would prefer `lon_coord` here!